db/data_migration/20160202102043_generate_content_ids_on_classifications.rb

```diff
@@ -0,0 +1,5 @@
+require 'securerandom'
+
+Classification.where(content_id: nil).each do |classification|
+  classification.update_column(:content_id, SecureRandom.uuid)
```

Comment:
FYI, there's a [rake task that does this](https://github.com/alphagov/whitehall/blob/master/lib/tasks/publishing_api.rake#L42), so we could potentially skip this altogether...